bin/bundle

```diff
@@ -39,7 +39,10 @@ m = Module.new do
 
   def gemfile
     gemfile = ENV["BUNDLE_GEMFILE"]
-    return gemfile if gemfile.present
```

Comment:
I think you can use `present?` here, with the extra `?` at the end, to avoid the error